python/tk_framework_desktopserver/errors.py

```diff
@@ -9,12 +9,26 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 """
-All custom exceptions that the desktop integration can emit are here.
+All custom exceptions that the browser integration can emit are here.
 """
 
 
-class MissingCertificate(Exception):
+class BrowserIntegration(Exception):
```

Comment:
We should follow the Python convention of having exception classes end in Error, so MissingCertificateError, PortBusyError, etc.